sentinel5dl/__init__.py

```diff
@@ -89,17 +90,41 @@ def __http_request(path, filename=None):
     logger.debug(f'Requesting {url}')
     with open(filename, 'wb') if filename else io.BytesIO() as f:
         curl = pycurl.Curl()
+        if ca_info:
+            curl.setopt(pycurl.CAINFO, ca_info)
         curl.setopt(curl.URL, url.encode('ascii', 'ignore'))
 
         curl.setopt(curl.USERPWD, f'{USER}:{PASS}')
         curl.setopt(curl.WRITEDATA, f)
         curl.setopt(curl.FAILONERROR, True)
 
+        # Use a Certificate Authority (CA) bundle if set
         if ca_info:
             curl.setopt(pycurl.CAINFO, ca_info)
 
-        curl.perform()
+        # abort if data transfer is slower than 30 bytes/sec during 60 seconds
+        curl.setopt(pycurl.LOW_SPEED_TIME, 60)
+        curl.setopt(pycurl.LOW_SPEED_LIMIT, 30)
```

Comment:
I'm still not sure why we would want to do that but maybe we can discuss this in person tomorrow.

Reply:
as i said earlier: the idea of this is to retry if the download connection is not responding but didn't errored. but I'm looking forward to clarify that personally